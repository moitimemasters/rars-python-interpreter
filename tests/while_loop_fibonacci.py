a = 1
b = 1

n = 0
while n < 8:
    c = a + b
    a = b
    b = c
    n += 1
a
