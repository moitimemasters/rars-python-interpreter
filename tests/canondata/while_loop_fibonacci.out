file: tests/while_loop_fibonacci.py
started lexing
ended lexing
started parsing ast
ended parsing ast
started compiling
compiling node: 8
compiling node: 0
compiling node: 8
compiling node: 0
compiling node: 8
compiling node: 0
compiling node: 27
compiling node: 6
compiling node: 5
compiling node: 0
compiling node: 8
compiling node: 6
compiling node: 5
compiling node: 5
compiling node: 8
compiling node: 5
compiling node: 8
compiling node: 5
compiling node: 8
compiling node: 5
compiling node: 0
compiling node: 5
ended compiling
compiled instructions:
	 Instruction: 2
	 Instruction: 8
	 Instruction: 2
	 Instruction: 8
	 Instruction: 2
	 Instruction: 8
	 Instruction: 0
	 Instruction: 2
	 Instruction: 7
	 Instruction: 6
	 Instruction: 0
	 Instruction: 0
	 Instruction: 7
	 Instruction: 8
	 Instruction: 0
	 Instruction: 8
	 Instruction: 0
	 Instruction: 8
	 Instruction: 0
	 Instruction: 2
	 Instruction: 7
	 Instruction: 8
	 Instruction: 5
	 Instruction: 13
	 Instruction: 0
started interpreting
ended interpreting
Interpretation result: 34
